dex/networks/ltc/params.go

```diff
@@ -18,34 +18,54 @@ var (
 		},
 	}
 	// MainNetParams are the clone parameters for mainnet.
-	MainNetParams = btc.ReadCloneParams(&btc.CloneParams{
```

Comment:
I was having a really hard time seeing my way to using the `CloneParams` stuff here.  I'd rather do this directly and transparently.

Reply:
`CloneParams` was a reduction of the `Params` to only the fields that were actually used, in order to simplify and document implementation of clones. The entire dex Bitcoin RPC implementation only needs those five fields. Someone trying to copy this clone impl. will now end up doing more work than they need, which is not trivial when there is no btcd clone from which to copy the parameters.

Reply:
Great points.  I'll switch back.  What started me in this direction was partly looking to make the `Addresser` used for server fees generic enough for clones, which means the HD fields.  Also the Name struck me as one that is reasonable to have set to something for logging and/or debugging.  Will address the fee address issues subsequently and update `CloneParams` at that time.

Reply:
Please do improve as you see necessary. Just wanted to convey the intention. I probably should have doc'd better.